.github/workflows/ci.yml

```diff
@@ -18,6 +16,8 @@ jobs:
       - name: Use Node.js 16
         uses: actions/setup-node@v3
         with:
+          cache: 'npm'
+          cache-dependency-path: packages/*/package-lock.json
```

Comment:
the `cache-dependency-path` is not the same than the others, you meant rather `package-lock.json` ?